tests/unit_return_type/mod.rs

```diff
@@ -0,0 +1,246 @@
+use crate::{test_lint_diagnostics, test_lint_fixer};
+
+const SIMPLE_RETURN_UNIT_TYPE: &str = r#"
```

Comment:
[nit]: I would much prefer reading this copied over and over and seeing the transformation result next to the input

Reply:
The only reason we do this way it's because the same code sample is used in the diagnostics test and fixer. But i agree, sometimes it's annoying to scroll to very top to see the code. Will make a task for rearranging code samples so each of them will be just above testing functions